main.py

```diff
@@ -1,6 +1,11 @@
 """ToDo."""
 from tasks.kyu8 import example
+from tasks.kyu8 import miles_per_gallon_to_kilometers_per_liter
 
+def make_conversion():
+    """Run conversion from miles per gallon into kilometers per liter."""
+    miles_per_g = input("miles per imperial gallon: ")
```

Comment:
Add check (try...except) and rewrite input. Also rewrite (_if '__name__'=='__main__'_)